File: leetcode-solutions/1848-sum-of-unique-elements/solution.py
class Solution(object):
    def sumOfUnique(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        freq, sum = defaultdict(int), 0
        for i in nums:
            freq[i] += 1
        for j in freq:
            if freq[j] == 1:
                sum += j
        return sum

File: leetcode-solutions/1848-sum-of-unique-elements/solution.java
class Solution {
    public int sumOfUnique(int[] nums) {
        HashMap <Integer, Integer> freq = new HashMap <> ();
        int sum = 0;
        for(int i : nums)
            freq.put(i, freq.getOrDefault(i,0) + 1);
        for(int key : freq.keySet()) {
            if(freq.get(key) == 1)
                sum += key;
        }
        return sum;
    }
}
